package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "proxy_photos",
+  "version": "1.0.0",
+  "description": "Package file for item detail page proxy server",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
script to run your server?

Reply:
Totally forgot about it, thanks for the catch!